### PR TITLE
change labels.json to new format and fixed tests

### PR DESCRIPTION
**Pull request by zdeveloper**
# Pull Request

## Description
- change `labels.json` to new format that can accommodate type
- fix broken benchmark tests, these broke when we started sending the confidence back

## Related Issues
#211

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [ ] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [ ] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [ ] I have updated the documentation, if applicable.
- [ ] I have added or updated test cases to cover my changes, if applicable.
- [ ] I have minimized the number of reviewers to include only those essential for the review.
- [ ] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: OCR/tests/assets/labels.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
-{
-  "255,0,0": "nbs_patient_id",
-  "0,0,255": "nbs_cas_id"
-}
\ No newline at end of file
+[
+  {
+    "color": "255,0,0",
+    "label": "nbs_patient_id"
```
Comment (schreiaj) [inline OCR/tests/assets/labels.json:4]: Should these have a `type` field as well?

File: OCR/ocr/api.py [hunk 2]
```diff
@@ -22,10 +22,10 @@ async def health_check():
 
 @app.post("/image_to_text/")
 async def image_to_text(source_image: UploadFile, segmentation_template: UploadFile, labels: str = Form()):
-    source_image_np = np.fromstring(await source_image.read(), np.uint8)
+    source_image_np = np.frombuffer(await source_image.read(), np.uint8)
```
Comment (schreiaj) [inline OCR/ocr/api.py:25]: Is this relevant to the labels.json or is it just more accurate to the type of data being sent?

**Review by schreiaj — APPROVED** [review]

File: OCR/ocr/api.py (hunk 2 above)
Comment (zdeveloper) [reply OCR/ocr/api.py:25]: It's not relevant, but I noticed that `fromstring` was deprecated and they recommend switching to `frombuffer`

File: OCR/tests/assets/labels.json (hunk 1 above)
Comment (zdeveloper) [reply OCR/tests/assets/labels.json:4]: Good Catch! though unused right now, it should be there

Comment (zdeveloper) [reply OCR/tests/assets/labels.json]: Fixed, thanks

**Pull request merged**